lib/pbench/test/unit/server/test_user_auth.py

```diff
@@ -298,6 +298,33 @@ def test_update_user(client, server_config):
             assert response.status_code == 200
             assert data_login["message"] == "Successfully logged in."
 
+    @staticmethod
+    def test_external_token_update(client, server_config):
```

Comment:
Does this test cover the case where a warning would be issued?

Reply:
No, this one is for covering case where user is attempting to update the auth_token externally. There already is a separate test case for testing the re-login with auth header that is `test_user_relogin` function

Reply:
How does that test verify the new behavior here?

Reply:
There isn't a new behavior per se in the login, I just removed the redundant auth token check when Authorization header is present during login request. 

This test is just proof check that we shouldn't be allowing the external auth_token update